kong/db/migrations/core/007_140_to_200.lua

```diff
@@ -1,6 +1,14 @@
 return {
   postgres = {
-    up = [[]],
+    up = [[
+      DO $$
+      BEGIN
+        ALTER TABLE IF EXISTS ONLY "routes" ADD COLUMN "path_handling" TEXT;
```

Comment:
we have not used `ADD COLUMN` on previous migrations, but just `ADD`
```suggestion
        ALTER TABLE IF EXISTS ONLY "routes" ADD "path_handling" TEXT;
```

Just for sake of consistency. I am with that `COLUMN` there too.